src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/InitializeDampedHarmonic.cpp

```diff
@@ -245,19 +251,25 @@ void InitializeDampedHarmonic<Dim, UseRollon>::DampedHarmonicCompute<Frame>::
         const tnsr::aa<DataVector, Dim, Frame>& spacetime_metric,
         const tnsr::aa<DataVector, Dim, Frame>& pi,
         const tnsr::iaa<DataVector, Dim, Frame>& phi,
-        const tnsr::I<DataVector, Dim, Frame>& coords, const double sigma_r,
-        const std::array<double, 3>& amplitudes,
-        const std::array<int, 3>& exponents) noexcept {
+        const tnsr::I<DataVector, Dim, Frame>& coords,
+        const GeneralizedHarmonic::gauges::DhGaugeParameters<Frame, false>&
+            parameters) noexcept {
   if (UNLIKELY(h_and_d4_h->number_of_grid_points() != get(lapse).size())) {
     h_and_d4_h->initialize(get(lapse).size());
   }
   damped_harmonic(
-      make_not_null(&get<Tags::GaugeH<Dim, Frame>>(*h_and_d4_h)),
-      make_not_null(&get<Tags::SpacetimeDerivGaugeH<Dim, Frame>>(*h_and_d4_h)),
+      make_not_null(
+          &get<GeneralizedHarmonic::Tags::GaugeH<Dim, Frame>>(*h_and_d4_h)),
+      make_not_null(
+          &get<GeneralizedHarmonic::Tags::SpacetimeDerivGaugeH<Dim, Frame>>(
+              *h_and_d4_h)),
       lapse, shift, spacetime_unit_normal_one_form, sqrt_det_spatial_metric,
-      inverse_spatial_metric, spacetime_metric, pi, phi, coords, amplitudes[0],
-      amplitudes[1], amplitudes[2], exponents[0], exponents[1], exponents[2],
-      sigma_r);  // indexes 0, 1, 2 refer to L1, L2, S terms, respectively
+      inverse_spatial_metric, spacetime_metric, pi, phi, coords,
+      parameters.amplitudes[0], parameters.amplitudes[1],
```

Comment:
I'm undecided on this suggestion/idea, so just looking for your thoughts. Something to think about: would it make the code cleaner to pass in just the options struct? I would expect not really :shrug:

Reply:
Yeah, a shrug is kind of where I'm at, too. I felt like the free function makes sense to have separate parameters, because you might imagine calling it without using the initialization infrastructure, and then having to use the struct might be annoying. I don't feel to strongly about it though

Reply:
Yea, let's leave as-is then! Thanks :)